lib/gollum-lib/committer.rb

```diff
@@ -53,8 +53,9 @@ def index
     # Returns a Gollum::Git::Actor.
     def actor
       @actor ||= begin
-        @options[:name]  = @wiki.default_committer_name if @options[:name].to_s.empty?
-        @options[:email] = @wiki.default_committer_email if @options[:email].to_s.empty?
+        name = @options[:name]
+        @options[:name]  = @wiki.default_committer_name if name.to_s.empty?
+        @options[:email] = @wiki.default_committer_email if name.to_s.empty?
```

Comment:
Err, sorry. Yes, I was actually thinking of a case where the user explicitly sets `:name` to `''` and `:email` to `'whatever'`. Then it will set `:email` to the default value, which I think is weird.

The question is also whether setting an empty name should be allowed or not. If we want to allow it we should just change the lines to:
```ruby
@options[:name] = @wiki.default_committer_name if @options[:name].nil?
@options[:email] = @wiki.default_committer_email if @options[:email].nil?
```
Not sure what to think about setting empty names though.